README.md

```diff
@@ -49,6 +49,11 @@ logs.)
 
 * `version`: Application version. Included in the `details.log.gz` file.
 
+* `label`: Label to attach to the github issue, and include in the details file.
+
+  If using the JSON upload encoding, this should be encoded as a `labels` field,
```

Comment:
Oh I see, each form part represents 1 label. I guess it's easier to do it like that than add in string splitting code.